rdm/init_files/documents/software_plan.md

```diff
@@ -93,16 +94,22 @@ To the extent possible, checking against these standards should be performed in
 ## Testing Plan
 
 [[62304:5.5.2]]
+All final tests must include the git hash or other objective reference that can be used to identify the exact software tested [[62304:5.1.11]].
 
 TODO: Write out a testing plan for {{ system.project_name }}.
 
 This plan should include a pass/fail criteria for the entire test suite.  E.g., you require that all unit tests pass and that all integration tests pass or the cause of the failure is understood and justified [[62304:5.7.1.a]]
+## Quality Assurance
```

Comment:
For consistency, I think we should surround the section titles with blank lines

Reply:
Yes. Makes scrolling navigation much easier!
Fixed.